src/app/components/EmbedConsentBanner/__snapshots__/index.test.tsx.snap

```diff
@@ -342,3 +342,172 @@ exports[`Embed Consent Banner should match snapshot 1`] = `
   </div>
 </div>
 `;
+
+exports[`Embed Consent Banner should match snapshot for TikTok embed 1`] = `
```

Comment:
I'm not sure how useful these snapshot tests are, they contain a lot of content, it might be worth checking for some specific elements being present instead e.g. https://github.com/bbc/simorgh/pull/10344/files#diff-916f8b0bda5fe11dbda29456caa21b1b1e8f50e176c45733f4d3ea35910fa7a7R478

Reply:
This is fair, I'll see if I can check for the common elements in the banner instead 👍🏼